front/src/applications/stdcm/hooks/useConflictsMessages.ts

```diff
@@ -12,7 +12,7 @@ const useConflictsMessages = (t: TFunction, selectedSimulationOutput?: StdcmSimu
   const [workConflicts, setWorkConflicts] = useState<string[]>([]);
 
   useEffect(() => {
-    if (!hasConflicts(selectedSimulationOutput)) return;
+    if (!hasConflicts(selectedSimulationOutput) || !selectedSimulationOutput.pathProperties) return;
```

Comment:
This change is not necessary anymore
```suggestion
    if (!hasConflicts(selectedSimulationOutput)) return;
```